src/pages/Swap/LimitOrderBox/components/CurrencySelectTooltip.tsx

```diff
@@ -0,0 +1,7 @@
+import { MouseoverTooltip } from '../../../../components/Tooltip'
+
+export const CurrencySelectTooltip = ({ children }: { children: React.ReactNode }) => (
+  <MouseoverTooltip content="Only available on WETH-pairs. Use WETH for limit orders." placement="top">
```

Comment:
This explanation is off(?) Essentially limit orders input token must be always an ERC20 token. I know COW is working on an extra step where ETH is wrapped and then deposited into a new contract, but nothing new on that front.

The current explanation is sufficient, but not technically correct. 

cc @0xVenky

Reply:
Yea, we will add a "Do you want to wrap" later on perhaps. Right now, even the balance is not fetched and makes it look incomplete. I tested this and gave feedback to Leo asking to show this tooltip only if a user has chosen XDAI or ETH as one of the token. The change is coming soon for that :)